server/package-lock.json

```diff
@@ -1,8 +1,8441 @@
 {
   "name": "sqlpad",
   "version": "6.7.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@JstDust Would it be possible to revert this lockfileVersion 2 change and keep it lockfileVersion 1? This might require installing npm 6 temporarily. If this is problematic I can pull into another branch, make changes, then merge.

Reply:
@rickbergfalk done